notebooks/book2/03/schools8.ipynb

```diff
@@ -0,0 +1,1684 @@
+{
```

Comment:
<p>does <code>arviz_trace_from_states()</code> from probml_utils not working? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/121/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>